clients/unshaded/src/main/java/tachyon/client/file/AbstractTachyonFileSystem.java

```diff
@@ -217,8 +218,18 @@ public boolean mount(TachyonURI tachyonPath, TachyonURI ufsPath, MountOptions op
     }
   }
 
```

Comment:
Now the `open` method is not in TachyonFileSystemCore, because only the `openIfExists` maps directly to an RPC method, so we're not overriding anything.